purchase_order_import/tests/test_order_response_import.py

```diff
@@ -426,11 +427,11 @@ def test_13(self):
             move_confirmed.note,
         )
         move_backorder = move_ids.filtered(
-            lambda s: s.state == "assigned" and s.product_qty == 3
+            lambda s: s.state == "draft" and s.product_qty == 3
```

Comment:
Here too, I am surprised that backorders in Odoo 14 are no longer confirmed automatically. Is this intentional on the side of Odoo?

Reply:
Ah, I see that Odoo now daisy chains a number of wizards when transferring pickings. It always creates the backorder, and depending on how the user clicks their way across those wizards, the pickings are either confirmed or cancelled. Maybe you can follow the picking confirmation flow in this test so that maybe you don't have to change what is actually tested.